scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerStateRecoverHelper.java

```diff
@@ -130,7 +130,7 @@ public RecoveredSchedulerState recover(long loadJobPeriod, RMProxy rmProxy) {
             }
         }
 
-        finishedJobs.addAll(dbManager.loadFinishedJobs(false, loadJobPeriod));
+        //        finishedJobs.addAll(dbManager.loadFinishedJobs(false, loadJobPeriod));
```

Comment:
We shouldn't do this but use the property instead

Reply:
Ok